hooks/code-sniffer.sh

```diff
@@ -7,10 +7,18 @@ then
   ./update-dependencies.sh $1
 fi
 
-# Install dependencies and configure phpcs
+robo setup:artifacts-directory
+
+# Install dependencies and configure phpcs.
 vendor/bin/phpcs --config-set installed_paths vendor/drupal/coder/coder_sniffer
 
 vendor/bin/phpmd modules/$1/src html cleancode,codesize,design,unusedcode --ignore-violations-on-exit --reportfile artifacts/phpmd/index.html
 vendor/bin/phpmetrics --extensions=php,inc,module --report-html=artifacts/phpmetrics --git modules/$1
-# Check coding standards
-vendor/bin/phpcs --standard=Drupal --report=junit --report-junit=artifacts/phpcs/phpcs.xml modules/$1
+
+# Check coding standards.
+if [ -f modules/$1/phpcs.xml.dist ]
```

Comment:
This is great and should we document this support in the README? Or does drupal.org support this now too?

Reply:
I don't think drupal.org supports it, no. I did this to get the node example module tests green, thought it would be useful for others. Will find a spot for it in the README, good call.